pkg/metrics/consume.go

```diff
@@ -0,0 +1,56 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+type ConsumerMetrics struct {
+	WrittenSamples            prometheus.Counter
```

Comment:
I think it probably make sense to have this metric as a Vector with a per tenant value as well

Reply:
Will do